specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2022-05-01/appconfiguration.json

```diff
@@ -812,58 +816,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppConfiguration/configurationStores/{configStoreName}/keyValues": {
```

Comment:
Why we remove this operation?
This is not only a breaking change for the swagger and all SDKs, but also breaks the ARM linter rule that a resource must have a list operation on its parent